README.md

```diff
@@ -1,3 +1,5 @@
+<details>
```

Comment:
Please leave top title and description of original document at outside the `<details>` tag.

Reply:
Ya i have made the changes, using a markdown formatter extension